sorc/ncep_post.fd/CLDRAD.f

```diff
@@ -2190,23 +2197,24 @@ SUBROUTINE CLDRAD
                 end do
                 !-- end of search 2
 
-                zceil = min(zceil1,zceil2) ! choose lower of zceil1 and zceil2
+!                zceil = min(zceil1,zceil2) ! choose lower of zceil1 and zceil2
```

Comment:
Is zceil2 from the summation principle? Is that why it's ignored?

Reply:
@joeolson42 Yes, the preceding lines define zceil2 which is from the summation principle.